README.md

```diff
@@ -317,13 +317,20 @@ Grape will also automatically respond to HEAD and OPTIONS for all GET, and just
 If you want to use ActiveRecord within Grape, you will need to make sure that ActiveRecord's connection pool
 is handled correctly.
 
+Rails 4
+
 The easiest way to achieve that is by using ActiveRecord's `ConnectionManagement` middleware in your
 `config.ru` before mounting Grape, e.g.:
-
 ```ruby
 use ActiveRecord::ConnectionAdapters::ConnectionManagement
+```
 
-run Twitter::API
+Rails 5+
+
+Due to ActiveRecord's `ConnectionManagement` was removed in Rails 5, so you can use [otr-activerecord](https://github.com/jhollinger/otr-activerecord), and add like below:
```

Comment:
It's not Grape's place to explain what happened in AR :) Just say `Use otr-activerecord as follows.`

Reply:
Nice suggestion